src/Components/UserFormWithAuth/Form.tsx

```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
+import { userInfo } from "./types";
+
+export const Form = () => {
+    const [useFormInfo, setUserFormInfo] = useState<userInfo>({
+        email: "",
+        password: "",
+        age: 0,
+        about: "",
+    });
+    const { email, password, age, about } = useFormInfo;
+
+    return (
+        <form>
+            <label>
+                <h5>Email: </h5>
+                <input
+                    value={email}
+                    onChange={(e) => {
+                        const email = e.target.value;
+                        setUserFormInfo({
+                            ...useFormInfo,
+                            email,
+                        });
+                    }}
+                />
```

Comment:
each input should have been its own file dealing with the state of only that area.